File: password_generator.py
from random import choice as c
from msvcrt import getch as g

def getch():
    """Lee un solo carácter de la entrada estándar."""
    return g().decode('utf-8')

def password_generator(num):
    # ΑαΔδΗηΚκΒβΕεΘθΛλΓγΖζΙιΜμΝνΠπ ΤτΧΞξΡρΥυΨΟοΣςσΦφΩχабвгдеёжзийклмнопрстуфхцчшщыюяъьэАБВГДЕЁЖЗИЙКЛМНОПРСТУФХЦЧШЩЫЮЯЪЬЭაბგდევზთიკლმნოპჟრსტუფქღყშჩცძწჭხჯჰあいうえおかきくけこさしすせそたちつってとなにぬねのはひふへほまみむめもやゆよらりるれろわゐゑをんがぎぐげござじずぜぞだぢづでどばびぶべぼぱぴぷぺぽアイウエオカキクケコサシスセソタチツっテトナニヌネノハヒフヘホマミムメモヤユヨラリルレロワヰヱヲンガギグゲゴザジズゼゾダヂヅデドバビブベボパピプペポ
    chars = "abcdefghijklmnñopqrstuvwxyzABCDEFGHIJKMNÑOPQRSTUVWXYZ!#$%&/)='?¡¿\+*~}]`{([^<>-_.:,;|°¬"
    password = ""
    for i in range(num):
        password += c(chars)
    print(f"Your password is: {password}")

def main():
    opc = 'Y'
    while opc != 'N' and opc != 'n':
        num=int(input("Write the num of characters: "))
        password_generator(num)
        print("Continue? Y/N ")
        opc = getch().upper()
        print()
    print("Presiona cualquier tecla y despues enter para salir . . . ")
    getch()

if __name__=='__main__':
    main()

File: password_generator.cpp
#include <iostream>
#include <locale>
#include <conio.h>
#include <string>
#include <random>

using namespace std;

string password_generator(int num){
	string chars = "abcdefghijklmnñopqrstuvwxyzABCDEFGHIJKMNÑOPQRSTUVWXYZ!#$%&/()='?¡¿+*~}]`{[^<>-_.:,;|°¬";
	string password;
	mt19937 rng(random_device{}());
	uniform_int_distribution<int> dist(0, chars.size() - 1);
	for (int i = 0; i < num; ++i) {
        password += chars[dist(rng)];
    }
    return password;
}

int main(){
	setlocale(LC_ALL, "spanish");
	
	char opc = 'Y';
    while (opc != 'N' && opc != 'n') {
        int num;
        cout << "Write the num of characters: ";
        cin >> num;
        string generated_password = password_generator(num);
        cout << "Your password is: " << generated_password << endl;
        cout << "Continue? Y/N ";
        cin >> opc;
	}
	
	cout << "Presiona una tecla para continuar..." << endl;
    _getch();
    
	return 0;
}
